app/src/main/java/com/example/kotlincalculator/mvp/view/ActivityView.kt

```diff
@@ -0,0 +1,14 @@
+package com.example.kotlincalculator.mvp.view
+
+import android.app.Activity
+import android.content.Context
+import java.lang.ref.WeakReference
+
+open class ActivityView(activity: Activity) : Activity() {
+    private val activityRef: WeakReference<Activity> = WeakReference(activity)
+    val activity: Activity
+        get() = activityRef.get()!!
```

Comment:
In this case can be used "?" instead of "!!" ?

Reply:
done